File: 4168-mirror-distance-of-an-integer/mirror-distance-of-an-integer.go
func mirrorDistance(n int) int {
    return abs(reverse(n)-n)
}

func reverse(x int)int{
    s:=0
    for x>0{
        s=s*10 + x%10
        x/=10
    }
    return s
}

func abs(x int)int{
    if x<0{
        return -x
    }
    return x
}

File: 4168-mirror-distance-of-an-integer/mirror-distance-of-an-integer.py
class Solution:
    def mirrorDistance(self, n: int) -> int:
        return abs(self.reverse(n)-n)

    def reverse(self, x: int)-> int:
        s=0
        while x>0:
            s=s*10 + x%10
            x //= 10
        return s